Realm/RLMObjectBase.mm

```diff
@@ -133,24 +106,48 @@ + (RLMObjectSchema *)sharedSchema {
     return RLMSchema.sharedSchema[self.className];
 }
 
-- (NSArray *)linkingObjectsOfClass:(NSString *)className forProperty:(NSString *)property {
-    if (!_realm) {
+void RLMObjectBaseSetRealm(__unsafe_unretained RLMObjectBase *object, __unsafe_unretained RLMRealm *realm) {
```

Comment:
:+1: that's been on my list of things to do, but it will only check when a literal `nil` is passed through